exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetWriter.java

```diff
@@ -40,6 +40,16 @@
 public class ParquetWriter extends AbstractWriter {
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ParquetWriter.class);
 
+/** Version of Drill's Parquet writer. Increment this version (by 1) any time we make any format change to the file.
+ * Format changes include
+ * 1) supporting new data types,
+ * 2) changes to the format of data fields,
+ * 3) adding new metadata to the file footer, etc.
+ * Newer readers must be able to read old files. The Writer version tells the Parquet reader how to interpret fields
```

Comment:
This is Javadoc, so needs HTML formatting. For the list:

    <ul>
    <li>Supporting new data types,</li>
    ...
    </ul>